src/main/java/cz/student/simon8491/TaskManager.java

```diff
@@ -1,9 +1,142 @@
 package cz.student.simon8491;
 
+import cz.common.ConsoleColors;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
 public class TaskManager {
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
+    private static String[][] tasks;
+    private static Scanner scanner;
+
+    public static void main(String[] args) {
+
+        tasks = loadDataToTab("tasks.csv");
```

Comment:
CLEAN-CODE: zvolil bych vice popisny nazev metody z ktereho je hned patrne o co jde.
```suggestion
        tasks = loadTasks("tasks.csv");
```